### DISCUSSION TITLE
How to transfer a Variable to a ArrayVariableAux  component

### DISCUSSION THREAD

**Discussion by snugook1108** [idaholab/moose#21577] [Q&A General] [answered]
How to transfer a Variable to a ArrayVariableAux  component using MultiApp. 

I would like to ask you about a Variable to transfer to a ArrayVariableAux component using MultiApp.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You will indeed want to transfer to a regular variable then use the BuildArrayVariableAux to move it to the array variable
https://mooseframework.inl.gov/source/auxkernels/BuildArrayVariableAux.html
We dont have support for transfers to and from array variables directly currently, though by the end of the summer it ll be increased a lot

Guillaume
